docs/source/guides/Data-sharing.md

```diff
@@ -0,0 +1,76 @@
+# Data Sharing
+
+## Background
+
+Data is key to neuroscience, and it should be shared for maximal utility. There is increasing pressure from funders and publishers to share data alongside publications:
+
+**Wellcome:**
+
+_“We expect our researchers to maximise the availability of research data, software and materials with as few restrictions as possible. As a minimum, the data underpinning research papers should be made available to other researchers at the time of publication, as well as any original software that is required to view datasets or to replicate analyses.”_
```

Comment:
Minor stylistic-pedantic issue. For this and following quotes, we could use the "official" markdown quote syntax with ">", for example:
```md
**Niko:**
> nothing useful
```
I think it will render nicely with Sphinx

Reply:
Thanks!